src/app.js

```diff
@@ -82,6 +80,7 @@ export default function createApp(rootElement) {
   form.appendChild(inputElement);
 
   const addButton = createBtn("add", "Add");
+  addButton.setAttribute("data-testid", "add-button");
```

Comment:
the Add button could be found by its name `Add` as this is the only UI element with this exact text